tasks/selinux.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: "Allow nginx httpd_sys_content_t to access the well-known root"
+  sefcontext:
+    target: '{{ certbot_well_known_root }}(/.*)?'
+    setype: httpd_sys_content_t
+    state: present
+  register: update_fcontexts
+  when: certbot_well_known_root is defined
+
+- name: Apply new SELinux file context to filesystem
+  command: "restorecon -irv {{ certbot_well_known_root }}"
+  when: update_fcontexts.changed
```

Comment:
I believe `update_fcontexts is changed` is the preferred syntax now, but also, is there a reason to do this as a task instead of a handler?

Reply:
Yeah, it *has* to run before the certbot step. We could make it a handler and flush it right? But I'd imagine doing that if/when we have more selinux steps that could invoke this.

Reply:
Any suggestions for making this into a handler? I could make a `selinux_controlled_paths` variable with the `certbot_well_known_root` in it, but in other places you usually define a list of paths, here it might have to be dynamically generated based on which paths have selinux restrictions applied to them?

Reply:
Whoops sorry for not following up and thanks for the details. Keeping it as a task is probably better then as long as there aren't other places it'd want to be notified from.

Reply:
Aight, just gonna merge this then